packages/react-integration/demo-app-ts/src/components/demos/TopologyDemo/components/StyleNode.tsx

```diff
@@ -138,24 +145,32 @@ const StyleNode: React.FunctionComponent<StyleNodeProps> = ({
 
   const LabelIcon = passedData.labelIcon;
   return (
-    <DefaultNode
-      element={element}
-      {...rest}
-      {...passedData}
-      dragging={dragging}
-      regrouping={regrouping}
-      showLabel={detailsLevel === ScaleDetailsLevel.high && showLabel}
-      showStatusBackground={detailsLevel === ScaleDetailsLevel.low}
-      showStatusDecorator={detailsLevel === ScaleDetailsLevel.high && passedData.showStatusDecorator}
-      onContextMenu={data.showContextMenu ? onContextMenu : undefined}
-      contextMenuOpen={contextMenuOpen}
-      labelIcon={LabelIcon && <LabelIcon noVerticalAlign />}
-      attachments={
-        detailsLevel === ScaleDetailsLevel.high && renderDecorators(element, passedData, rest.getShapeDecoratorCenter)
-      }
-    >
-      {detailsLevel !== ScaleDetailsLevel.low && renderIcon(passedData, element)}
-    </DefaultNode>
+    <Layer id={hover ? TOP_LAYER : DEFAULT_LAYER}>
```

Comment:
This is causing every hovered node to swap to the top layer. I don't believe that was intended behavior?

Reply:
It was intended. Without this when the node is enlarged, there can be other nodes that intersect and draw over portions of the enlarged node.

Reply:
but even when at full scale, the same hover behavior is new.